AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift

```diff
@@ -10,47 +10,101 @@ import Foundation
     }
 
     convenience public init(callKitEnabled: Bool) {
-        self.init(audioMode: .stereo48K, callKitEnabled: callKitEnabled, enableAudioRedundancy: true, videoMaxResolution: VideoResolution.videoResolutionHD)
+        self.init(audioMode: Self.defaultAudioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: callKitEnabled,
+                  enableAudioRedundancy: Self.defaultEnableAudioRedundancy,
+                  videoMaxResolution: Self.defaultVideoMaxResolution)
     }
 
     convenience public init(enableAudioRedundancy: Bool) {
-         self.init(audioMode: .stereo48K, callKitEnabled: false, enableAudioRedundancy: enableAudioRedundancy, videoMaxResolution: VideoResolution.videoResolutionHD)
+        self.init(audioMode: Self.defaultAudioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: Self.defaultCallKitEnabled,
+                  enableAudioRedundancy: enableAudioRedundancy,
+                  videoMaxResolution: Self.defaultVideoMaxResolution)
     }
 
     convenience public init(videoMaxResolution: VideoResolution) {
-        self.init(audioMode: .stereo48K, callKitEnabled: false, enableAudioRedundancy: true, videoMaxResolution: videoMaxResolution)
+        self.init(audioMode: Self.defaultAudioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: Self.defaultCallKitEnabled,
+                  enableAudioRedundancy: Self.defaultEnableAudioRedundancy,
+                  videoMaxResolution: videoMaxResolution)
     }
 
     convenience public init(audioMode: AudioMode, callKitEnabled: Bool) {
-        self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: true, videoMaxResolution: VideoResolution.videoResolutionHD)
+        self.init(audioMode: audioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: callKitEnabled,
+                  enableAudioRedundancy: Self.defaultEnableAudioRedundancy,
+                  videoMaxResolution: Self.defaultVideoMaxResolution)
     }
 
     convenience public init(audioMode: AudioMode, callKitEnabled: Bool, enableAudioRedundancy: Bool) {
-        self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: enableAudioRedundancy, videoMaxResolution: VideoResolution.videoResolutionHD)
+        self.init(audioMode: audioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: callKitEnabled,
+                  enableAudioRedundancy: enableAudioRedundancy,
+                  videoMaxResolution: Self.defaultVideoMaxResolution)
     }
 
-    public init(audioMode: AudioMode, callKitEnabled: Bool, enableAudioRedundancy: Bool, videoMaxResolution: VideoResolution) {
```

Comment:
I think we still need to keep this init, since it has been published before.

Reply:
Thanks for pointing that out, I somehow missed it